<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Streaming</class>
 <widget class="QWidget" name="Streaming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="streamingFrame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1051</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QLabel" name="streamingLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>671</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);
background:rgba(243,115,33,0.2);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>Streaming Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateTimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 12pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);
text-align: center; /* 수평 중앙 정렬 */</string>
    </property>
    <property name="text">
     <string>Date - Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="licensetitleLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 13pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>LicensePlate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
        color:  rgb(243,115,33);
        background:white;
        border:1px solid rgb(243,115,33);
        font: 500 10pt &quot;Quicksand Medium&quot;;
}

QPushButton:hover {
        color:  rgb(255, 255, 255);
        background-color:rgba(243,115,33,0.3);
        border:1px solid rgb(243,115,33);
        font: 500 8pt &quot;Quicksand Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
        color:  rgb(243,115,33);
        background:white;
        border:1px solid rgb(243,115,33);
        font: 500 10pt &quot;Quicksand Medium&quot;;
}

QPushButton:hover {
        color:  rgb(255, 255, 255);
        background-color:rgba(243,115,33,0.3);
        border:1px solid rgb(243,115,33);
        font: 500 8pt &quot;Quicksand Medium&quot;;
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color:  rgb(154, 154, 154);
        background:white;
        border:1px solid rgb(243,115,33);
        font: 500 10pt &quot;Quicksand Medium&quot;;
</string>
    </property>
    <property name="text">
     <string>RTSP Server IP Address</string>
    </property>
   </widget>
   <widget class="QFrame" name="licensePlateframe">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(243,115,33);
background:white;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="number1">
       <property name="styleSheet">
        <string notr="true">background:none;
border:1px solid rgb(243,115,33);
font: 300 bold 13pt &quot;Quicksand Light&quot;;
color:  rgb(243,115,33);</string>
       </property>
       <property name="text">
        <string>Vehicle2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="stateArea">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(243,115,33);
background:white;
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="stateFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="gate_state">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>131</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border:1px dashed rgbrgb(162, 162, 162);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="alarm_state">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:white;
border:1px dashed rgbrgb(162, 162, 162);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>Customer Information</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="VehicleEntry">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 500 13pt &quot;Quicksand Medium&quot;;
background-color: rgba(243,115,33,0.5);
border:1px solid rgb(243,115,33);</string>
    </property>
    <property name="text">
     <string>Vehicle Entry</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
